lib/many_points.dart

```diff
@@ -0,0 +1,18 @@
+/// The visualise library.
+library many_points;
+
+import 'package:image/image.dart';
+import 'dart:math' as math;
+import 'dart:io' as io;
+
+export 'package:image/image.dart' show Color;
+part 'src/visualisation.dart';
+part 'src/range.dart';
+
+/// Returns [num x, num y, num value]
+typedef List DataTransformFunction(
+    int x, int y, num value, Range xRange, Range yRange, Range dataRange);
+
+/// Returns the color of the point
+typedef int ColorTransformFunction(
```

Comment:
Because the image package uses an int to represent a colour to reduce memory allocations. The Color class is used only as a namespace for color operations.

https://github.com/brendan-duncan/image/blob/master/lib/src/color.dart